README.md

```diff
@@ -34,73 +34,66 @@ from aicsimageio import AICSImage, imread
 
 # Get an AICSImage object
 img = AICSImage("my_file.tiff")
-img.data  # returns 6D STCZYX numpy array
-img.dims  # returns string "STCZYX"
-img.shape  # returns tuple of dimension sizes in STCZYX order
-
-# Get 6D STCZYX numpy array
+img.data  # returns 5D TCZYX numpy array
+img.dims.order  # returns string "TCZYX"
+img.dims.X  # returns size of X dimension
+img.shape  # returns tuple of dimension sizes in TCZYX order
+img.get_image_data("CZYX", T=0)  # returns 4D CZYX numpy array
+
+# Change scene
+img.set_scene(1)
```

Comment:
img.scenes # returns a list of scene ids found in the file, which can then be used to loop or enumerate for set_scene